spec/openapi.yaml

```diff
@@ -963,6 +980,91 @@ components:
           type: string
           example: "/dev/tty"
 
+    StatusJob:
+      type: object
+      description: Telemetry info about current job, all values are optional
+      properties:
+        id:
```

Comment:
[question]
What is that ID? Is it possible to have there path to the file that is being printing?

Reply:
Sure, I'll add path